driver-core/src/main/java/com/datastax/driver/core/SimpleStatement.java

```diff
@@ -193,6 +194,19 @@ public Object getObject(int i) {
     }
 
     /**
+     * Returns the named values as a {@code Map<String, Object>}.
+     *
+     * @return the named values of this statement.
+     * @throws IllegalStateException     if this statement does not have named values.
+     */
+    public Map<String, Object> getObjectMap() {
```

Comment:
I have done the changes in a local branch, couldn't help my self:-). I think it looks good, let me know if you want me to update the PR.